backend/src/main/java/ca/bc/gov/backendstartapi/enums/SeedlotStatus.java

```diff
@@ -0,0 +1,26 @@
+package ca.bc.gov.backendstartapi.enums;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+/** Possible status for Seed lots. */
+@Schema(description = "The possible status for a seed lot.")
+public enum SeedlotStatus implements DescribedEnum {
```

Comment:
I think we should keep the consistence, calling this enumeration `SeedlotStatusEnum`.

Reply:
Renamed.